data/MIMIC_CXR.py

```diff
@@ -0,0 +1,132 @@
+from skimage.io import imread, imsave
+import numpy as np
+import os,sys,os.path
+import pandas as pd
+import torch
+import torch.utils.data
+
+
+class MIMICCXR_Dataset:
+    """
+
+    Dataset website here:
+    https://physionet.org/content/mimic-cxr-jpg/2.0.0/
+    """
+
+    def __init__(self, imgpath, csvpath, metacsvpath, views=["PA"], transform=None, data_aug=None,
```

Comment:
mutable default argument here will create memory leak:

https://docs.python-guide.org/writing/gotchas/#what-you-wrote